DatabaseBuddy.Core/Extender/NumericExtender.cs

```diff
@@ -16,33 +16,12 @@ public static long ToLongValue(this object Value, long DefaultValue)
       }
       catch { return DefaultValue; }
     }
-
-    public static long ToLongValue(this long Value)
-    {
-      return Value;
-    }
-
-    public static long ToLongValue(this long? Value)
-    {
-      return !Value.HasValue ? 0 : Value.Value;
-    }
     public static long ToLongValue(this object Value)
     {
       return ToLongValue(Value, 0);
     }
 
     #region [ToDouble]
-    public static double ToDoubleValue(this double Value)
-    {
-      return Value;
-    }
-    public static double ToDoubleValue(this double? Value)
```

Comment:
The ToDoubleValue with a nullable long is a valid method and should not be removed because this could be needed quite often.